src/pages/CurrenciesListPage.jsx

```diff
@@ -1,11 +1,25 @@
-import React from "react";
+import React, { useState } from "react";
 import Heading from "../components/Heading";
+import Select from "../components/Selector";
 
 function CurrenciesList() {
+  const options = [
+    { value: "elo", label: "elo" },
+    { value: "siema", label: "siema" },
+  ];
+  const [value, setValue] = useState("");
   return (
     <div>
       <Heading variant="title">Currency Converter</Heading>
       <Heading variant="subtitle">Choose base currency</Heading>
+      <Select
+        selectedValue={value}
```

Comment:
It's a bad practice to rename the default HTML attribute names when providing props for them, please revert the change and rename the `useState` instead
```ts
const [selectedValue, setSelectedValue] = useState("");
```

Reply:
done